### PR TITLE
2017 Event

### PR DESCRIPTION
**Pull request by csexton**
Update the website for the 2017 event. This should include project suggestions, attendee details and anything else related to our upcoming conference.

### CODE REVIEW

**Comment by csexton** [comment]
One thing we need to do is re-enable the google form (located [here](https://docs.google.com/forms/d/1M3PJepMOZcqUcIN81Ju7YEeXpQKhYWnSWnmgeZTX33w/closedform#start=openform)) for scholarship applications. @seanmarcia is that something you can do?

**Comment by seanmarcia** [comment]
Added it back in.

**Comment by seanmarcia** [comment]
👍

File: 2017.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+---
+
+### Ruby for Good 2017
+
+[Join us on Slack](https://rubyforgood.herokuapp.com/) before, during, and after the event for announcements and making friends.
+
+###### Thursday June 16th
```
Comment (seanmarcia) [inline 2017.md]: May 18th

File: 2017.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+The location is only accessible by car — we are helping to [coordinate carpooling](https://docs.google.com/forms/d/1aA0fJbbmxYW3lTqP6A7zLOnZbECzCd5d9f7L9ok_jLI/viewform?c=0&w=1). Please fill out the form if you need a ride or can offer a ride and Kalimar will do his best to match people up! Contact Kalimar at [kalimar@rubyforgood.org](mailto:kalimar@rubyforgood.org).
+
+If you arrive and the gate is locked, dial #90 at the phone at gate #2 for access.
+
+Once you enter through the gates, drive to the top of the hill (the Smithsonian Tower with yellow and blue sun will be on your right). Make a right after the Smithsonian tower, and parking is on the right.
+
+After arriving, head to the “Residences” building to check in.
+
+### FONZ Tours
+
```
Comment (seanmarcia) [inline 2017.md:59]: Maybe just have:

The [Friends of the National Zoo (FONZ)](https://nationalzoo.si.edu/JoinFonz/join.cfm) is giving us a personalized, not-open-to-the-public tour of the Smithsonian Conservation Biology Institute (SCBI) grounds and animal habitats so we can learn about red pandas, bison, and other friendly creatures, their homes, and the research being done on-site.

File: index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
     </nav>
     <div class="hero-content text-center">
       <div class="hero-small">
-        <hr /><span>June 16th - June 19th, 2016</span><hr />
+        <hr /><span>May 17th - May 20th, 2017</span><hr />
```
Comment (seanmarcia) [inline index.html:17]: 18

File: 2017.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+* 6:30pm - Werewolf, board games, socializing, fun!
+
+###### Saturday June 18th
+
+* 8:00 am - Breakfast
+* 9:00 am - Work on projects
+* 12:00 pm - Lunch
+* 1:00 pm - Work on projects
+* 5:00 pm - Dinner
+
```
Comment (pollygee) [inline 2017.md:36]: Should we add more board games / socializing here?

File: 2017.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+### Ruby for Good 2017
+
+[Join us on Slack](https://rubyforgood.herokuapp.com/) before, during, and after the event for announcements and making friends.
+
+###### Thursday June 16th
+
+* Arrive before 5:00 pm — most of the organizing team is planning on arriving at 10:00 am but feel free to beat us there!
+* 1:00 pm - 26 people on [FONZ](https://nationalzoo.si.edu/JoinFonz/join.cfm) tour of Smithsonian-Mason Institute (see below) — [reserve your spot](https://docs.google.com/spreadsheets/d/1gTi3wUj8MJLG637wdYBsasBrca53P7X-FFjXWw-SF2k/edit?usp=sharing).
```
Comment (pollygee) [inline 2017.md:12]: We will need to create a new sign up or fix the old one.  If you want to just clear out the old google doc, I can clear it out to get it ready for a new signup.

Have the dates and times been confirmed for the tours?

File: 2017.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* 5:00 pm - Event kickoff and announcements
+* 5:30 pm - Team pitches, team assignments
+* 6:30 pm - Dinner, initial team discussion, setting up environments, cloning down repos, etc.
+* 8:30 pm? - Board games, computer games, socializing, and other random fun.
```
Comment (pollygee) [inline 2017.md:16]: Why do we have a '?' here, will some of the teams keep working past 8:30?

File: 2017.md (hunk 5 above)
Comment (seanmarcia) [reply 2017.md:12]: I don't think we're going to list any tour times in the initial schedule or have a signup.

Comment (seanmarcia) [reply 2017.md:12]: I think instead we're just going to say "If you are on Team X,Y,Z and you want to do the tour, you are taking it at 1pm, etc" Several of the teams found it pretty disruptive that random people would disappear for the tour. I think if we break it down by team, then they will know when people will be missing.

File: 2017.md (hunk 6 above)
Comment (seanmarcia) [reply 2017.md:16]: I think it is supposed to reference 8:30 or earlier... Whenever everyone has their environments/repos ready to start the next day.

**Pull request merged**